include/CoolPropTools.h

```diff
@@ -616,50 +624,39 @@ template<class T> void normalize_vector(std::vector<T> &x)
         std::replace( file_path.begin(), file_path.end(), '\\', '/'); // replace all '\' with '/'
         
         #if defined(__ISWINDOWS__)
-        char sep = '\\';
+        const char sep = '\\'; // well, Windows (and DOS) allows forward slash "/", too :)
```

Comment:
But some functions in WINAPI do not accept forward slashes as the path delimiter.  I'm not sure if that is the case within this function.  I'd be happy to change this if we can show it works with /